.hashibot.hcl

```diff
@@ -1500,3 +1500,43 @@ behavior "pull_request_path_labeler" "service_labels" {
     ]
   }
 }
+
+behavior "regexp_issue_labeler" "panic_label" {
```

Comment:
Default hashibot behaviors from the terraform-providers org, no defaults are set in the HC org.